swagger2.0/oic.sec.sdi.swagger.json

```diff
@@ -0,0 +1,171 @@
+            "204": {
+              "description" : "The SDI is updated."
+            }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "interface" : {
+      "in" : "query",
+      "name" : "if",
+      "type" : "string",
+      "enum" : [ "oic.if.baseline", "oic.if.rw" ]
+    }
+  },
+  "definitions": {
+    "Sdi" : {
+      "properties": {
+        "uuid": {
+          "description": "A UUID that identifies the Security Domain, set by DOTS during onboarding.",
+          "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+          "type": "string"
+        },
+        "name": {
+          "description": "Human-friendly name for the Security Domain, set by DOTS during onboarding.",
+          "type": "string"
+        },
+        "rt": {
+          "description": "Resource Type of the Resource.",
+          "items": {
+            "maxLength": 64,
+            "type": "string",
+            "enum": ["oic.r.sdi"]
+          },
+          "minItems": 1,
+          "readOnly": true,
+          "type": "array"
+        },
+        "n": {
+          "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/n"
+        },
+        "id": {
+          "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/id"
+        },
+        "priv": {
+          "description": "Flag to indicate whether the Security Domain Information is copied to “/oic/res”, and thus, whether it is publicly visible or private.",
+          "type": "boolean"
+        },
+        "if" : {
+          "description": "The interface set supported by this Resource.",
+          "items": {
+            "enum": [ "oic.if.baseline", "oic.if.rw" ],
+            "type": "string"
+          },
+          "minItems": 1,
+          "readOnly": true,
+          "type": "array"
+        }
+      },
+      "type" : "object",
+      "required": [ "uuid", "name", "priv" ]
+    },
+    
+    "Sdi-Update" : {
+      "properties": {
+        "uuid": {
+          "description": "A UUID that identifies the Security Domain, set by DOTS during onboarding.",
+          "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+          "type": "string"
+        },
+        "name": {
+          "description": "Human-friendly name for the Security Domain, set by DOTS during onboarding.",
+          "type": "string"
+        },
+        "rt": {
+          "description": "Resource Type of the Resource.",
+          "items": {
+            "maxLength": 64,
+            "type": "string",
+            "enum": ["oic.r.sdi"]
+          },
+          "minItems": 1,
+          "readOnly": true,
+          "type": "array"
+        },
+        "n": {
+          "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/n"
+        },
+        "id": {
```

Comment:
"id" should not be in the UPDATE schema.

Reply:
Removed